yapapi/rest/activity.py

```diff
@@ -32,6 +41,8 @@ async def new_activity(self, agreement_id: str) -> "Activity":
 
 
 class Activity(AsyncContextManager["Activity"]):
+    """Higher-level wrapper for REST's Activity endpoint"""
```

Comment:
We call it "mid-level" in the docstring for `rest/__init__.py`, maybe let's stick with the term?

Reply:
```suggestion
    """Mid-level wrapper for REST's Activity endpoint"""
```